profile/README.md

```diff
@@ -3,10 +3,11 @@
 ![ET](https://user-images.githubusercontent.com/5240896/142377175-2a02f457-6168-4838-b228-e6f3ff54866b.png)
 
 
-The Einstein Telescopehttp is an advanced gravitational-wave observatory, currently in the planning stage. 
+The Einstein Telescope is an advanced gravitational-wave observatory, currently in the planning stage. 
 
 Gravitational waves contain information about the most extreme events in the universe, from the nature of black holes and neutron stars to the first moments after the Big Bang. Thanks to these waves, we can study the cosmos as never before.
 
 ### Links
 - [Einstein Telescope](http://www.et-gw.eu/index.php)
 - [EMR](https://www.einsteintelescope.nl/en/?set_lang=en) - The Euregio-Meuse-Rhine area.
+- 
```

Comment:
Maybe we can add a section on how someone can become a collaborator? 

Or maybe adding a section of the people to contact if they need something?

Reply:
Very good ideas. I'll add a section on guidelines for collaborators and how to become one.

I'll add a section with links for each of the candidate sites and the relevant contact people in each site.